includes/admin/setting-admin.php

```diff
@@ -73,6 +75,24 @@ function openWindowContent () {
                 ],
             ];
 
+            $newSetting[] = [
```

Comment:
@brunoferreiralkn Linha 31 todo o conteúdo do atributo $logContent precisa ser em JSON mesmo se for apenas a mensagem de 'Nenhum spam bloqueado'. Isso gera um erro de tipagem no javascript impedindo que a página seja carregada corretamente caso não existam logs.